internal/pkg/mapper/orchestrator/remote_load_test.go

```diff
@@ -0,0 +1,415 @@
+						ConfigId:    `123`,
+						Content: utils.PairsToOrderedMap([]utils.Pair{
+							{
+								Key: `task`,
+								Value: *utils.PairsToOrderedMap([]utils.Pair{
+									{Key: `mode`, Value: `run`},
+								}),
+							},
+							{Key: `continueOnFailure`, Value: false},
+							{Key: `enabled`, Value: true},
+						}),
+					},
+					{
+						Name:        `Task 3`,
+						ComponentId: `foo.bar2`,
+						ConfigId:    `789`,
+						Content: utils.PairsToOrderedMap([]utils.Pair{
+							{
+								Key: `task`,
+								Value: *utils.PairsToOrderedMap([]utils.Pair{
+									{Key: `mode`, Value: `run`},
+								}),
+							},
+							{Key: `continueOnFailure`, Value: false},
+							{Key: `enabled`, Value: false},
+						}),
+					},
+				},
+			},
+			{
+				Key:       model.PhaseKey{Index: 1},
+				DependsOn: []model.PhaseKey{{Index: 0}},
+				Name:      `Phase With Deps`,
+				Content:   utils.NewOrderedMap(),
+				Tasks: []model.Task{
+					{
+						Name:        `Task 2`,
+						ComponentId: `foo.bar2`,
+						ConfigId:    `456`,
+						Content: utils.PairsToOrderedMap([]utils.Pair{
+							{
+								Key: `task`,
+								Value: *utils.PairsToOrderedMap([]utils.Pair{
+									{Key: `mode`, Value: `run`},
+								}),
+							},
+							{Key: `continueOnFailure`, Value: false},
+							{Key: `enabled`, Value: true},
+						}),
+					},
+				},
+			},
+		},
+	}, internalObject.Orchestration)
+}
+
+func TestMapAfterRemoteLoadWarnings(t *testing.T) {
+	t.Parallel()
+	context, logs := createMapperContext(t)
+
+	contentStr := `
+{
+  "phases": [
+    {
+      "id": 123,
+      "name": "Phase",
+      "dependsOn": []
+    },
+    {
+      "id": 456
+    },
+    {}
+  ],
+  "tasks": [
+    {
+      "id": 1001,
+      "name": "Task 1",
+      "phase": 123,
+      "task": {
+        "componentId": "foo.bar1",
+        "configId": "123",
+        "mode": "run"
+      }
+    },
+    {
+      "id": 1002,
+      "name": "Task 2",
+      "phase": 789,
+      "task": {
+        "componentId": "foo.bar2",
+        "configId": "456",
+        "mode": "run"
+      }
+    },
+    {}
+  ]
+}
+`
+
+	content := utils.NewOrderedMap()
+	json.MustDecodeString(contentStr, content)
+	configKey := model.ConfigKey{
+		BranchId:    123,
+		ComponentId: model.OrchestratorComponentId,
+		Id:          `456`,
+	}
+	apiObject := &model.Config{ConfigKey: configKey, Content: content}
+	internalObject := apiObject.Clone().(*model.Config)
+	recipe := &model.RemoteLoadRecipe{ApiObject: apiObject, InternalObject: internalObject}
+
+	// Invoke
+	assert.Empty(t, apiObject.Relations)
+	assert.Empty(t, internalObject.Relations)
+	assert.NoError(t, NewMapper(context).MapAfterRemoteLoad(recipe))
+
+	// Warnings
+	expectedWarnings := `
+WARN  Warning: invalid orchestrator config "branch:123/component:keboola.orchestrator/config:456":
+  - missing phase[1] "name" key
+  - missing phase[2] "id" key
+  - phase "789" not found, referenced from task[1] "Task 2"
```

Comment:
Ak najdeme nejak nezrovnalosti v `remote` state -> tak sa vypise `warning`.
Nechceme uzivatela pre jeden blby orchestrator uplne stopnut, kedze `remote` stav moze opravit napr. tak, ze si to `pullne/opravi/pushne`.